src/components/Cities.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+
+const Cities = ({ city, aqi }) => {
+  let weatherStatus = '';
+
+  if (aqi === 1) {
+    weatherStatus = 'Good';
+  } else if (aqi === 2) {
+    weatherStatus = 'Fair';
+  } else if (aqi === 3) {
+    weatherStatus = 'Moderate';
+  } else if (aqi === 4) {
+    weatherStatus = 'Poor';
+  } else if (aqi === 5) {
+    weatherStatus = 'Very Poor';
+  }
```

Comment:
- [ ] **[OPTIONAL]** I suggest that you create a separate utility function for this piece of logic, for example `getWeatherStatusFromAqi`:

```js
const getWeatherStatusFromAqi = (aqi) => {
  if (aqi === 1) {
    return 'Good';
  } else if (aqi === 2) {
    return 'Fair';
  } else if (aqi === 3) {
    return 'Moderate';
  } else if (aqi === 4) {
    return 'Poor';
  } else {
    return 'Very Poor';
  }
}
```

and then, in your component, you can simply call this function:

```js
// src/components/Cities.js
import PropTypes from 'prop-types';

const Cities = ({ city, aqi }) => {
  const weatherStatus = getWeatherStatusFromAqi(aqi);
  ...
}
```

Ending up with a cleaner component and separating the logic from the UI! 🤓 🔧